.gitpod.yml

```diff
@@ -22,8 +22,8 @@ tasks:
     yarn download:sample-traces &&
     yarn
   command: >
-    export GITPOD_URL=$(gp url 8080) &&
-    export TRACE_SERVER_URL=$(echo "$GITPOD_URL/tsp/api") &&
+    export WORKSPACE_URL=$(gp url 8080) &&
+    export TRACE_SERVER_URL=$(echo "$WORKSPACE_URL/tsp/api") &&
```

Comment:
Could we not use "GITPOD_WORKSPACE_URL"?

![image](https://user-images.githubusercontent.com/28311615/134421305-98ec5119-a03f-423d-a48b-12baa485232c.png)
Source: [Gitpod environment variables documentation](https://www.gitpod.io/docs/environment-variables)